.github/workflows/prod-deploy.yml

```diff
@@ -0,0 +1,51 @@
+name: Beta Deployment Workflow
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  beta-deployment:
+    if: >-
+      github.event.issue.pull_request &&
+      contains(github.event.comment.body, 'Approve Prod Deployment')
+    name: Get approval for deployment
```

Comment:
Name should be beta deployment workflow?

Reply:
Sure, Will Change to avoid confusion.